lab_guide_CN.md

```diff
@@ -47,8 +47,16 @@ go version go1.16.4 linux/amd64
 ```
 git clone https://github.com/vldbss-2021/vldb-labs-yourid.git .
```

Comment:
```suggestion
git clone git@github.com/vldbss-2021/vldb-labs-yourid.git .
```

It's not able to clone private repo via https URL without authentication, and the support for authentication by password is removed now.
https://github.blog/2020-12-15-token-authentication-requirements-for-git-operations/